DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: Rblpapi
 Title: R Interface to 'Bloomberg'
-Version: 0.3.8
+Version: 0.3.9
```

Comment:
That '+ 0.0.1' increment is for release.  Please make it 0.3.8.1 to make as first interim version based off 0.3.8.

Reply:
Changed.